production.cfg

```diff
@@ -109,6 +109,9 @@ zcml-additional =
         ${buildout:zcml-additional-fragments}
     </configure>
 
+early-zcml =
+zcml =
```

Comment:
Should we also define a `buildout:instance-zcml` like we did for `instance-eggs`? This could work for prod (`instance0`) as well as for dev (`instance`).
I usually prefer not to set options of the isntance parts whenever possible. I think it would be great to standardize that.